app/src/main/java/com/hedvig/app/ui/compose/composables/appbar/TopAppBar.kt

```diff
@@ -0,0 +1,92 @@
+package com.hedvig.app.ui.compose.composables.appbar
```

Comment:
Nothing changed in this file. Just made some overloads to easily have a top app bar with a close button and a back button. They function the same